.github/workflows/containers.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and upload Docker image
+
+on:
+  push:
+    tags:
+      - "*.*.*"
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Do you want to build for every PR to branch main?

Reply:
Yeah, I was thinking that it could make integration testing easier, if we want to do that before committing to merge new PRs.